File: 12933.java
import java.util.*;
class Solution {
    public long solution(long n) {
        String answer = "";
        String[] array = String.valueOf(n).split("");
        Arrays.sort(array);
        for(int i = array.length-1 ; i > -1 ; i--){
            answer += array[i];
        }
        return Long.parseLong(answer);
    }
}

File: 12933.swift
func solution(_ n:Int64) -> Int64 {
    var str = String(n)
    var arraylist = [String]()
    for i in 0..<str.count{
        let index = str.index(str.startIndex , offsetBy: i)
        arraylist.append(String(str[index]))
    }
    arraylist.sort()
    arraylist.reverse()
    var answer = ""
    for i in 0..<arraylist.count{
        answer.append(arraylist[i])
    }
    // print(answer)
    return Int64(answer)!
}
